core/src/debug/frameInfo.cpp

```diff
@@ -41,23 +45,37 @@ void FrameInfo::endUpdate() {
 
 void FrameInfo::beginFrame() {
 
-    if (getDebugFlag(DebugFlags::tangram_infos)) {
+    if (getDebugFlag(DebugFlags::tangram_infos) || getDebugFlag(DebugFlags::stats)) {
         s_startFrameTime = clock();
     }
 
 }
 
-void FrameInfo::draw(const View& _view, TileManager& _tileManager) {
 
-    if (getDebugFlag(DebugFlags::tangram_infos)) {
+void FrameInfo::draw(const View& _view, TileManager& _tileManager, float _pixelsPerPoint) {
+
+    if (getDebugFlag(DebugFlags::tangram_infos) || getDebugFlag(DebugFlags::stats)) {
         static int cpt = 0;
 
+        static std::deque<float> cputime;
+        static std::deque<float> rendertime;
```

Comment:
Will keep `std::deque` because for array access (https://github.com/tangrams/tangram-es/pull/790/commits/483679b7c78a43e3137f99023359c9b593d57884).